deno/payloads/common.ts

```diff
@@ -58,3 +58,27 @@ export const PermissionFlagsBits = {
 Object.freeze(PermissionFlagsBits);
 
 export type LocalizationMap = Partial<Record<LocaleString, string | null>>;
+
+/**
+ * https://discord.com/developers/docs/topics/opcodes-and-status-codes#json
+ */
+export interface DiscordAPIError {
```

Comment:
Sorry!! After some internal thoughts, it'd be better to drop the Discord prefix (as its *obviously* for Discord)